include/fmt/format.h

```diff
@@ -1718,8 +1718,9 @@ auto find_escape(const Char* begin, const Char* end)
 
 inline auto find_escape(const char* begin, const char* end)
     -> find_escape_result<char> {
-  if (const_check(!use_utf8)) return find_escape<char>(begin, end);
-  auto result = find_escape_result<char>{end, nullptr, 0};
+  auto result = find_escape<char>(begin, end);
```

Comment:
is not eliding the 24 byte copy slower than unconditionally calling `find_escape<char>`?